### ISSUE TITLE
Select the first plan by default

### ISSUE THREAD

**Issue by htmgarcia** [OSTraining/Simplerenew#386]
When multiple plans selection is enabled, you need to include *?select=some-plan* in the url

**Comment by billtomczak** [comment]
Yes - that was by design. Shouldn't be too much trouble to change.
